vendor/extensions/announcements/app/models/refinery/announcements/announcement.rb

```diff
@@ -35,6 +36,13 @@ def tag_list=(titles)
       def tag_content_type
         self.tags.where(tag_type: "content_type")[0].title
       end
+
+      def tags_length
+        if tags.length < 2
```

Comment:
Why are you checking if it's at least two? Is there a default tag that we add the topic tags onto?

Reply:
Yeah it uses a tag for category as well as one for content type. It gets a default content type tag in the hidden field. Maybe not the ideal implementation but I'm living with it for now.